scripts/generateAssetData/baseAssets.ts

```diff
@@ -11,7 +11,7 @@ export const ethereum: AssetWithNetworkName = {
   networkName: 'Ethereum',
   precision: 18,
   color: '#5C6BC0',
-  icon: 'https://assets.coincap.io/assets/icons/256/eth.png',
+  icon: '',
```

Comment:
update all references to ethereum icon to the one suggested here please 
```suggestion
  icon: 'https://coin-images.coingecko.com/coins/images/279/large/ethereum.png?1696501628',
```

Reply:
Fixed in [d50ae4f](https://github.com/shapeshift/web/pull/7298/commits/d50ae4fbe0b84892069e74dfc28a17a7599c9e16)